src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/HistoricalBeneficialOwnerValidatorTest.java

```diff
@@ -205,6 +214,21 @@ void testErrorReportedWhenCeasedDateIsInTheFuture() {
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
+    @Test
+    void testErrorReportedWhenCeasedDateIsInTheFutureAndAndIsUnableToObtainAllTrustInfoIsSetToTrue() {
```

Comment:
same with this one, perhaps add another test where unable to obtain is explicitly false